symbology/create-symbols-from-web-styles/src/main/java/com/esri/samples/create_symbols_from_web_styles/CreateSymbolsFromWebStylesSample.java

```diff
@@ -0,0 +1,316 @@
+      uniqueValueRenderer.getFieldNames().add("cat2");
+
+      // set the unique value renderer on the feature layer
+      featureLayer.setRenderer(uniqueValueRenderer);
+
+      // create a symbol style from a web style
+      symbolStyle = new SymbolStyle("Esri2DPointSymbolsStyle", null);
+
+      // display an error if the symbol style fails to load
+      symbolStyle.addDoneLoadingListener(() -> {
+        if (symbolStyle.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
+          new Alert(Alert.AlertType.ERROR, "Error: could not load symbol style. Details: \n"
+            + symbolStyle.getLoadError().getMessage()).show();
+        }
+      });
+
+      // setup the UI for the legend
+      setupLegend();
+
+      // create a list of the required symbol names from the web style
+      ArrayList<String> symbolNames = new ArrayList<>();
+      symbolNames.addAll(Arrays.asList("atm", "beach", "campground", "city-hall", "hospital", "library", "park",
+        "place-of-worship", "police-station", "post-office", "school", "trail"));
+
+      for (String symbolName : symbolNames) {
+
+        // search for each symbol in the symbol style
+        List<String> searchKey = new ArrayList<>();
+        searchKey.add(symbolName);
+        ListenableFuture<Symbol> searchResult = symbolStyle.getSymbolAsync(searchKey);
+        searchResult.addDoneListener(() -> {
+          try {
+            // get the symbol from the search result
+            Symbol symbol = searchResult.get();
+
+            // get a list of all categories to be mapped to the symbol
+            List<String> categories = mapSymbolNameToField(symbolName);
+
+            for (String category : categories) {
+
+              // create a unique value for each category
+              List<Object> categoryValue = new ArrayList<>();
+              categoryValue.add(category);
+              UniqueValueRenderer.UniqueValue uniqueValue = new UniqueValueRenderer.UniqueValue(
+                "", symbolName, symbol, categoryValue);
+
+              // add each unique value to the unique value renderer
+              uniqueValueRenderer.getUniqueValues().add(uniqueValue);
+            }
+
+            // create and add an image view and a label for the symbol to the legend on the UI
+            ImageView imageView = createImageView(symbol);
+            Label gridPaneLabel = new Label(symbolName);
+            gridPane.add(imageView, 0, symbolNames.indexOf(symbolName) + 2);
+            gridPane.add(gridPaneLabel, 1, symbolNames.indexOf(symbolName) + 2);
+
+          } catch (Exception e) {
+            // on any error, display the stack trace.
+            e.printStackTrace();
+          }
+        });
+      }
+
+      // add a map scale changed listener on the map view to control the symbol sizes at different scales
+      mapView.addMapScaleChangedListener(mapScaleChangedEvent -> {
+        featureLayer.setScaleSymbols(!(mapView.getMapScale() < 80000));
+      });
```

Comment:
There's two things we can improve here: since it's a single line within the lambda, we can get rid of the curly braces and the `;` at the end of that single line. Also, instead of doing `!(x < y)`, we can just do (x >= y). I means one less set of parentheses, and the `!` is a bit easier to miss than seeing `>=` 
```suggestion
      mapView.addMapScaleChangedListener(mapScaleChangedEvent ->
              featureLayer.setScaleSymbols(mapView.getMapScale() >= 80000));
```